api/src/main/java/com/epam/pipeline/manager/cluster/NatGatewayManager.java

```diff
@@ -229,9 +228,7 @@ private boolean processScheduledTermination(final Service service, final Map<Int
                 messageHelper.getMessage(MessageConstants.NAT_ROUTE_REMOVAL_PORT_REMOVAL_FAILED));
         }
         if (activePorts.containsKey(port)
-            && !kubernetesManager.refreshCloudPipelineServiceDeployment(tinyproxyServiceName,
-                                                                        DEPLOYMENT_REFRESH_RETRIES,
-                                                                        DEPLOYMENT_REFRESH_TIMEOUT_SEC)) {
+            && !kubernetesManager.refreshDeployment(null, tinyproxyServiceName, tinyproxyLabelSelector)) {
```

Comment:
Let's add an overloaded `kubernetesManager.refreshDeployment` method without the first argument.

Reply:
Done